contracts/AllDay.cdc

```diff
@@ -610,13 +610,12 @@ access(all) contract AllDay: NonFungibleToken {
         }
     }
 
-    access(all) entitlement ManageBadges
-
     access(all) resource Badges {
-        access(all) var slugToBadge: {String: Badge}
-        access(all) var playIdToBadgeSlugs: {UInt64: {String: {String: String}}}
-        access(all) var editionIdToBadgeSlugs: {UInt64: {String: {String: String}}}
-        access(all) var momentIdToBadgeSlugs: {UInt64: {String: {String: String}}}
+        access(all) let slugToBadge: {String: Badge}
```

Comment:
`access(self)` should be enough for all the maps also

Reply:
ah, yes